### PR TITLE
[circle-partitioner] Read partition config ini

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable circle-partitioner to read partition config ini file.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6230, from draft #6231

File: compiler/circle-partitioner/src/PartitionRead.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+#include <luci/Log.h>
+
+#include <stdexcept>
+
+namespace
+{
+
+using namespace partee;
+
+const char *_section_partition = "partition";
```
Comment (mhs4670go) [inline compiler/circle-partitioner/src/PartitionRead.cpp:31]: Just curiosity, is there any reason to use `const char *` instead of `std::string`?

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/circle-partitioner/src/PartitionRead.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/circle-partitioner/src/PartitionRead.cpp:31]: for simplicity and didn't want to initialize `std::string` in binary load time :)

**Pull request merged**